src/natives/screenshot/win/Program.cs

```diff
@@ -330,7 +330,7 @@ struct WindowBorderOnScreenFlags {
             }
 
             IntPtr hwnd = (IntPtr)Convert.ToInt32(args[0]);
-            string browser = args[1];   //NOTE: "chrome" || "ie" || "firefox" || "opera"
+            string browser = args[1];   //NOTE: "chrome" || "iexplore" || "firefox" || "opera"
```

Comment:
we should keep this comment actual or just remove it. In the first case we should add the `processName` for the `MSEdge` browser